src/components/TheRegistration.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div class="login">
+    <v-form
+      v-on:submit.prevent="onSubmit"
+      ref="form"
+      v-model="valid"
+      lazy-validation
+    >
+      <v-text-field
+        v-model="username"
+        label="Benutzername"
+        placeholder="Benutzername"
+        :counter="maxUsername"
+        @input="$v.username.$touch()"
+        @blur="$v.username.$touch()"
+        :error-messages="usernameErrors"
+        box
+      ></v-text-field>
+      <v-text-field
+        v-model="email"
+        label="E-Mail"
+        placeholder="E-Mail"
+        @input="$v.email.$touch()"
+        @blur="$v.email.$touch()"
+        :error-messages="emailErrors"
+        box
+      ></v-text-field>
+      <v-text-field
+        v-model="password"
+        label="Passwort"
+        placeholder="Passwort"
+        type="password"
+        ref="password"
+        @input="$v.password.$touch()"
+        @blur="$v.password.$touch()"
+        :error-messages="passwordErrors"
+        box
+      ></v-text-field>
+      <v-text-field
+        v-model="passwordRepeat"
+        label="Passwort wiederholen"
+        placeholder="Passwort wiederholen"
+        type="password"
+        @input="$v.passwordRepeat.$touch()"
+        @blur="$v.passwordRepeat.$touch()"
+        :error-messages="passwordRepeatErrors"
+        box
+      ></v-text-field>
+
+      <v-btn type="button" @click="onSubmit" color="primary">
+        Registrieren
+      </v-btn>
+    </v-form>
+  </div>
+</template>
+
+<script>
+import user from "../_service/user.service";
+import { validationMixin } from "vuelidate";
+import {
+  required,
+  maxLength,
+  email,
+  sameAs,
+  helpers
+} from "vuelidate/lib/validators";
+const passwordRegex = helpers.regex(
+  "password",
+  /^(?=.*[A-Z])(?=.*[0-9])(?=.*[a-z])(?=.*[,.\-_#'+*~]).{8,}$/
+);
+
+let maxUsername = 15;
```

Comment:
Größe ich 120.
Können wir was sinnvolles finden, sowas nur einmal im Code zu definieren (Am besten im Core), was dann für alles gilt?

Reply:
Machen wir wenn auch in einem separaten Ticket.